src/pages/datasets-page/index.tsx

```diff
@@ -23,13 +23,9 @@ const delayTableLoad = async (promise: Promise<typeof import('./populated-table'
 };
 
 const DatasetsPage: FC = () => {
-  let pageSubtitle = 'Mangler tittel';
-  const datasetsContext = useDatasetsContext();
-
-  if (datasetsContext.datasets?.length && datasetsContext.datasets?.length > 0)
-    pageSubtitle = datasetsContext.datasets[0]?.publisher.name;
-
   const { catalogId } = useParams();
+  const datasetsContext = useDatasetsContext();
+  const pageSubtitle = datasetsContext.organizationName ?? localization.withoutTitle;
```

Comment:
Er det ikke bedre å vise orgnr i stedet?

Reply:
Kanskje begge deler men vi gjør det ikke i nye begrepskatalogen heller.

Reply:
Vi viser kun «Mangler tittel» i Alle kataloger-oversikten også. Og her vises ikke org.nr i URL slik det gjør nå man har valgt katalog. Her kunne det ha vært fint å se org.nr, men samtidig er det kanskje nyttig å også vite at navnet mangler. Og når man har valgt en katalog er vel ikke navnet lenger en tittel, da tittelen er f.eks. Datasettkatalog. Kanskje skulle det her vises «Mangler navn» i stedet for «Mangler tittel»? Og for API-katalogen så viser vi ikke virksomhetens navn - kun tittel Datatjenestekatalog.
![image](https://github.com/Informasjonsforvaltning/dataset-catalog-gui/assets/43232700/cf0748bb-d637-4268-991f-e090130368ab)
![image](https://github.com/Informasjonsforvaltning/dataset-catalog-gui/assets/43232700/8a4df30f-7a3a-409b-ab58-a97b12fc7951)
![image](https://github.com/Informasjonsforvaltning/dataset-catalog-gui/assets/43232700/bfdc152e-1912-405b-bba3-ddd4353c9b35)

Reply:
Foreslår at vi nå bruker «Mangler tittel» slik det er gjort ellers i løsningen. Og så får vi senere bli enig om dette skal endres, ev. hva som skal vises og så implementere dette for alle katalogene.